python/revng/internal/cli/_commands/fetch_debuginfo/common.py

```diff
@@ -26,10 +27,18 @@ def log_warning(self, message):
 log_warning = logger.log_warning
 
 
+session = requests.Session()
+adapter = requests.adapters.HTTPAdapter(
+    max_retries=Retry(total=5, backoff_factor=1.0, status_forcelist=[408, 429, 500, 502, 503, 504])
```

Comment:
How did you put together the HTTP status list? What are they?

At least add a comment.

Reply:
Added comments to the codes
Basically I compiled the list by looking at the list of error codes and adding the ones that "made sense", e.g. related to timeouts, throttling or the server being in some way unavailable.